client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -0,0 +1,190 @@
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import QuerySiteRoles from 'components/data/query-site-roles';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteRoles } from 'state/site-roles/selectors';
+import {
+	isJetpackModuleActive,
+	isJetpackModuleUnavailableInDevelopmentMode,
+	isJetpackSiteInDevelopmentMode
+} from 'state/selectors';
+
+class JetpackSiteStats extends Component {
+	static defaultProps = {
+		isSavingSettings: false,
+		isRequestingSettings: true,
+		fields: {}
+	};
+
+	static propTypes = {
+		handleAutosavingToggle: PropTypes.func.isRequired,
+		setFieldValue: PropTypes.func.isRequired,
+		isSavingSettings: PropTypes.bool,
+		isRequestingSettings: PropTypes.bool,
+		fields: PropTypes.object,
+	};
+
+	onChangeToggleGroup = ( groupName, fieldName ) => {
+		return () => {
+			const { setFieldValue } = this.props;
+			let groupFields = this.getCurrentGroupFields( groupName );
+
+			if ( groupFields.includes( fieldName ) ) {
+				groupFields = groupFields.filter( field => field !== fieldName );
+			} else {
+				groupFields.push( fieldName );
+			}
+
+			setFieldValue( groupName, groupFields, true );
+		};
+	};
+
+	getCurrentGroupFields( groupName ) {
+		const { fields } = this.props;
+
+		if ( ! fields[ groupName ] ) {
+			return [];
+		}
+		return fields[ groupName ];
+	}
+
+	renderToggle( name, label, checked = null, onChange = null ) {
+		const {
+			fields,
+			handleAutosavingToggle,
+			isRequestingSettings,
+			isSavingSettings,
+			moduleUnavailable,
+			statsModuleActive
+		} = this.props;
+
+		if ( checked === null ) {
+			checked = !! fields[ name ];
+		}
+
+		if ( onChange === null ) {
+			onChange = handleAutosavingToggle( name );
+		}
+
+		return (
+			<CompactFormToggle
+				checked={ checked }
+				disabled={ isRequestingSettings || isSavingSettings || moduleUnavailable || ! statsModuleActive }
+				onChange={ onChange }
+				key={ name }
+			>
+				{ label }
+			</CompactFormToggle>
+		);
+	}
+
+	render() {
+		const {
+			siteId,
+			siteRoles,
+			translate
+		} = this.props;
+		const header = translate( 'Collecting valuable traffic stats and insights' );
+
+		return (
+			<div className="site-settings__traffic-settings">
+				<QueryJetpackConnection siteId={ siteId } />
+				<QuerySiteRoles siteId={ siteId } />
+
+				<SectionHeader label={ translate( 'Site stats' ) } />
+
+				<FoldableCard
+					className="site-settings__foldable-card is-top-level"
+					header={ header }
+					clickableHeader
+				>
+					<FormFieldset>
+						<div className="site-settings__info-link-container">
+							<InfoPopover position={ 'left' }>
+								<ExternalLink href={ 'https://jetpack.com/support/wordpress-com-stats/' } target="_blank">
+									{ translate( 'Learn more about WordPress.com Stats' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Learn more about WordPress.com' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).